client/verta/verta/_registry/modelversion.py

```diff
@@ -74,7 +74,7 @@ def __repr__(self):
             "id: {}".format(msg.id),
             "registered model id: {}".format(msg.registered_model_id),
             "experiment run id: {}".format(msg.experiment_run_id),
-            "archived status: {}".format(msg.archived),
+            "archived status: {}".format(msg.archived == 1),
```

Comment:
Since this is a [`TernaryEnum`](https://github.com/VertaAI/modeldb/blob/0662c3c/protos/protos/public/registry/RegistryService.proto#L148), I prefer we reference it explicitly.
```suggestion
            "archived status: {}".format(msg.archived == _CommonCommonService.TernaryEnum.TRUE),
```